full-build.sh

```diff
@@ -55,6 +55,7 @@ while [ "$1" != "" ]; do
 done
 
 MVN_ARGS+=(-PuseJenkinsSnapshots)
+MVN_ARGS+=(-Dsurefire.rerunFailingTestsCount=2)
```

Comment:
Please do three attempts

Reply:
Is it possible to mark candidates for flakes or is it applied to all tests?

Reply:
I don't think that's possible to mark candidates. The documentation does not mention that. However, flakes are recorded.

After switching to 3 attempts can I merge?